frontend/package.json

```diff
@@ -6,10 +6,14 @@
     "@testing-library/jest-dom": "^5.11.9",
     "@testing-library/react": "^11.2.5",
     "@testing-library/user-event": "^12.8.3",
+    "axios": "0.21.1",
     "create-react-app": "^4.0.3",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-icons": "^4.2.0",
```

Comment:
yes for react icons 🎊

Reply:
i dont even know where this is coming from... it shouldnt hurt anyone